tools/protocol-schema-check/res/protocol_schema.toml

```diff
@@ -273,7 +273,7 @@ TransactionReceipt = 968816131
 TransactionV0 = 197396442
 TransactionV1 = 2594686420
 TransferAction = 1078380396
-TrieChanges = 3833039794
+TrieChanges = 2613580820
```

Comment:
Do we need any protocol upgrade or db upgrade handling?

Reply:
That shouldn't be necessary as the params we are adding/changing are not serialized.